docs_user/modules/proc_migrating-mon-from-controller-nodes.adoc

```diff
@@ -0,0 +1,411 @@
++
+----
+sudo cephadm shell -m /tmp/mon.yaml
+ceph orch apply -i /mnt/mon.yaml
+----
++
+[NOTE]
+The effect of applying the `mon.yaml` spec is to normalize the existng placement
+strategy to use `labels` instead of `hosts`. By doing this any node with the `mon`
+label is able to host a Ceph mon daemon.
+You can execute the step above once to avoid multiple iterations over this
+procedure when multiple Ceph Mons are migrated.
+
+. Check the status of the {CephCluster} and the orchestrator daemons list. Make
+  sure that the three mons are in quorum and listed by the `ceph orch` command:
++
+----
+# ceph -s
+  cluster:
+    id:     f6ec3ebe-26f7-56c8-985d-eb974e8e08e3
+    health: HEALTH_OK
+
+  services:
+    mon: 3 daemons, quorum oc0-controller-0,oc0-controller-1,oc0-controller-2 (age 19m)
+    mgr: oc0-controller-0.xzgtvo(active, since 32m), standbys: oc0-controller-1.mtxohd, oc0-controller-2.ahrgsk
+    osd: 8 osds: 8 up (since 12m), 8 in (since 18m); 1 remapped pgs
+
+  data:
+    pools:   1 pools, 1 pgs
+    objects: 0 objects, 0 B
+    usage:   43 MiB used, 400 GiB / 400 GiB avail
+    pgs:     1 active+clean
+----
++
+----
+[ceph: root@oc0-controller-0 /]# ceph orch host ls
+HOST              ADDR           LABELS          STATUS
+oc0-ceph-0        192.168.24.14  osd
+oc0-ceph-1        192.168.24.7   osd
+oc0-controller-0  192.168.24.15  _admin mgr mon
+oc0-controller-1  192.168.24.23  _admin mgr mon
+oc0-controller-2  192.168.24.13  _admin mgr mon
+----
+
+. On the source node, back up the `/etc/ceph/` directory. In case of issues,
+  this backup allows you to have the ability to execute cephadm and get a shell
+  to the {Ceph} cluster from the source node:
++
+----
+mkdir -p $HOME/ceph_client_backup
+sudo cp -R /etc/ceph $HOME/ceph_client_backup
+----
+
+. Before draining the source node and relocating the IP address of the storage
+  network to to the target node, fail the ceph-mgr if it is active on the
+  source node:
++
+----
+ceph mgr fail <mgr instance>
+----
+
+. Drain the source node and start the mon migration. From the cephadm shell,
+  remove the labels on the source node:
++
+----
+    for label in mon mgr _admin; do
+           ceph orch host rm label <source_node> $label;
+    done
+----
+
+. Remove the running mon daemon from the source node:
++
+----
+cephadm shell -- ceph orch daemon rm mon.<source_node> --force"
+----
+
+. Run the drain command:
++
+----
+cephadm shell -- ceph drain <source_node>
+----
+
+. Remove the `<source_node>` host from the {CephCluster}:
++
+----
+cephadm shell -- ceph orch host rm <source_node> --force"
+----
+
+* Replace `<source_node>` with the hostname of the source node.
+
+The source node is not part of the cluster anymore, and should not appear in
+the {Ceph} host list when running `cephadm shell -- ceph orch host ls`.
+However, a `sudo podman ps` in the source node might list both mon and mgr still
+running.
+
+----
+[root@oc0-controller-1 ~]# sudo podman ps
```

Comment:
```suggestion
----
+
* Replace `<source_node>` with the hostname of the source node.
+
The source node is not part of the cluster anymore, and should not appear in
the {Ceph} host list when running `cephadm shell -- ceph orch host ls`.
However, a `sudo podman ps` in the source node might list both mon and mgr still
running.
+
----
[root@oc0-controller-1 ~]# sudo podman ps
```

Reply:
Should line 194-210 be a note? We're letting customers know that they might still see mon and mgr still running in the source node even though they're not running?